vendor/vendor.json

```diff
@@ -1839,12 +1874,11 @@
 			"revisionTime": "2017-11-23T00:13:03Z"
 		},
 		{
-			"checksumSHA1": "k5TsfhX2vvfffYbZQ4fMKtL5+cA=",
+			"checksumSHA1": "STxYqRb4gnlSr3mRpT+Igfdz/kM=",
+			"origin": "github.com/elastic/beats/vendor/github.com/spf13/pflag",
 			"path": "github.com/spf13/pflag",
-			"revision": "583c0c0531f06d5278b7d917446061adc344b5cd",
-			"revisionTime": "2018-04-12T12:09:13Z",
-			"version": "v1.0.1",
```

Comment:
this shouldn't have an origin, we use this directly.  This is very confusing

Reply:
This is also vendored by beats, so I run a `govendor delete github.com/spf13/pflag` and then `make update-beats`, which was adding it as a beats dependency. I think we should have the same versions as beats wherever we use the same dependencies. 
However, I revert this change for now and we can discuss our vendoring approach for such situations.